internal/pkg/manifest/lb_fargate_manifest.go

```diff
@@ -29,7 +29,7 @@ type ImageWithPort struct {
 type LBFargateConfig struct {
 	RoutingRule      `yaml:"http,flow"`
 	ContainersConfig `yaml:",inline"`
-	Public           bool               `yaml:"public"`
+	Public           *bool              `yaml:"public"` // A pointer because we don't want the environment override to default to false.
```

Comment:
I don't know if we even need this parameter.

Reply:
me neither what would we expect if a user set this to false?

Reply:
hmm....that is confusing. Does this mean it wouldn't be a public Load Balancer, or that the task itself is not public?

Reply:
Removed the field!   
We can add it back later if we can come up with a use-case, right now we have a public loadbalancer and the service will receive requests from it.